Kaldi now requires python3 in addition to python2.7

Since recently, python3 is now required by Kaldi `check_dependencies.sh` script (see https://github.com/kaldi-asr/kaldi/commit/8ad898c2b8a506d10b9c2fb31949a148aed6844d).

Consequently, trying to build the image from scratch will fail at step 6 due to the fact that the `python3` package is not installed.
Adding `python3` to the list of installed packages is needed to fix this.

---

Error log:

```
Step 6/8 : RUN git clone https://github.com/kaldi-asr/kaldi &&     cd /opt/kaldi/tools &&     make &&     ./install_portaudio.sh &&     cd /opt/kaldi/src && ./configure --shared &&     sed -i '/-g # -O0 -DKALDI_PARANOID/c\-O3 -DNDEBUG' kaldi.mk &&     make depend && make &&     cd /opt/kaldi/src/online && make depend && make &&     cd /opt/kaldi/src/gst-plugin && make depend && make &&     cd /opt &&     git clone https://github.com/alumae/gst-kaldi-nnet2-online.git &&     cd /opt/gst-kaldi-nnet2-online/src &&     sed -i '/KALDI_ROOT?=\/home\/tanel\/tools\/kaldi-trunk/c\KALDI_ROOT?=\/opt\/kaldi' Makefile &&     make depend && make &&     rm -rf /opt/gst-kaldi-nnet2-online/.git/ &&     find /opt/gst-kaldi-nnet2-online/src/ -type f -not -name '*.so' -delete &&     rm -rf /opt/kaldi/.git &&     rm -rf /opt/kaldi/egs/ /opt/kaldi/windows/ /opt/kaldi/misc/ &&     find /opt/kaldi/src/ -type f -not -name '*.so' -delete &&     find /opt/kaldi/tools/ -type f \( -not -name '*.so' -and -not -name '*.so*' \) -delete &&     cd /opt && git clone https://github.com/alumae/kaldi-gstreamer-server.git &&     rm -rf /opt/kaldi-gstreamer-server/.git/ &&     rm -rf /opt/kaldi-gstreamer-server/test/
 ---> Running in a75ba361cf20
Cloning into 'kaldi'...
extras/check_dependencies.sh
extras/check_dependencies.sh: python3 is not installed
extras/check_dependencies.sh: we recommend that you run (our best guess):
 sudo apt-get install  
Makefile:31: recipe for target 'check_required_programs' failed
make: *** [check_required_programs] Error 1
The command '/bin/sh -c git clone https://github.com/kaldi-asr/kaldi &&     cd /opt/kaldi/tools &&     make &&     ./install_portaudio.sh &&     cd /opt/kaldi/src && ./configure --shared &&     sed -i '/-g # -O0 -DKALDI_PARANOID/c\-O3 -DNDEBUG' kaldi.mk &&     make depend && make &&     cd /opt/kaldi/src/online && make depend && make &&     cd /opt/kaldi/src/gst-plugin && make depend && make &&     cd /opt &&     git clone https://github.com/alumae/gst-kaldi-nnet2-online.git &&     cd /opt/gst-kaldi-nnet2-online/src &&     sed -i '/KALDI_ROOT?=\/home\/tanel\/tools\/kaldi-trunk/c\KALDI_ROOT?=\/opt\/kaldi' Makefile &&     make depend && make &&     rm -rf /opt/gst-kaldi-nnet2-online/.git/ &&     find /opt/gst-kaldi-nnet2-online/src/ -type f -not -name '*.so' -delete &&     rm -rf /opt/kaldi/.git &&     rm -rf /opt/kaldi/egs/ /opt/kaldi/windows/ /opt/kaldi/misc/ &&     find /opt/kaldi/src/ -type f -not -name '*.so' -delete &&     find /opt/kaldi/tools/ -type f \( -not -name '*.so' -and -not -name '*.so*' \) -delete &&     cd /opt && git clone https://github.com/alumae/kaldi-gstreamer-server.git &&     rm -rf /opt/kaldi-gstreamer-server/.git/ &&     rm -rf /opt/kaldi-gstreamer-server/test/' returned a non-zero code: 2
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -18,6 +18,7 @@ RUN apt-get update && apt-get install -y  \
     libtool-bin \
     make \
     python2.7 \
+    python3 \
     python-pip \
     python-yaml \
     python-simplejson \

